encoding/encoding.go

```diff
@@ -100,54 +89,32 @@ func (e *Encoder) UnpackMessage(encodedBody []byte) ([]byte, MessageType, error)
 	}
 
 	flags := binary.BigEndian.Uint64(decoded[:8])
-	message, err := rsa.DecryptOAEP(
-		sha256.New(),
-		rand.Reader,
-		e.ownPrivKey,
-		decoded[8:],
-		rsaLabel,
-	)
+	r, err := age.Decrypt(bytes.NewReader(decoded[8:]), e.ownPrivKey)
```

Comment:
I'm afraid of out of array. I'm not sure that there is any problem, but check could simplify debbuging for us in future.

Reply:
Good catch, fixed